components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/main/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/core/ServerClaimManagementService.java

```diff
@@ -1444,4 +1455,27 @@ private void validateAttributeMappings(List<AttributeMappingDTO> attributeMappin
             }
         }
     }
+
+    private void validateRootOrganization() throws ClaimMetadataClientException {
```

Comment:
Let's add a method comment saying, organizations except primary organizations. Btw the keyword is the root organization rather primary organization IIRC.

Reply:
`allowClaimManagementAccessForRootOrganizations()`

Kind of a method signature will convay the operation without reading the method body.

Reply:
`validateClaimModificationEligibility` / `validateClaimModificationAllowed`

Reply:
Fixed